client/pages/Register.jsx

```diff
@@ -16,12 +36,17 @@ const Register = ({ onMount }) => {
     axios.post(apiUrl, data)
       .then((response) => {
         console.log(response.data);
+        if (data.file && data.file[0]) {
+          const formData = new FormData();
+          formData.append('file', data.file[0]);
+          const uploadUrl = 'http://localhost:3000/marvel/upload';
+          axios.post(uploadUrl, formData);
```

Comment:
Oh, you're still posting as FormData. That should be easy to deal with on the backend using the exercises uploadMiddleware.